infrastructure/k8s/services/secrets.yaml

```diff
@@ -0,0 +1,29 @@
+project_name: "taskcluster-secrets"
+secrets:
+  azure_account: ".Values.tfAzureAccount"
+  azure_crypto_key: ".Values.azureCryptoKey"
+  azure_signing_key: ".Values.azureSigningKey"
+  azure_table_name: "Secrets"
+  force_ssl: "false"
+  monitoring_enable: "true"
+  node_env: "production"
+  publish_metadata: "false"
+  pulse_password: ".Values.pulsePassword"
+  pulse_username: ".Values.pulseUsername"
+  taskcluster_access_token: ".Values.secretsAccessToken"
```

Comment:
Can you add generating this to create-random-secrets.sh ?

Reply:
Added in https://github.com/sciurus/taskcluster/pull/19